engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -551,8 +551,8 @@ public void dispose() {
         renderableWorld.dispose();
         worldProvider.dispose();
         // TODO: Shift this to a better place, after a RenderGraph class has been implemented.
-        // TODO: Call ResetDefaultInstance() for every StateChange after it is a method of StateChange.
-        SetViewportToSizeOf.ResetDefaultInstance();
+        // TODO: Call disposeDefaultInstance() for every StateChange after it is a method of StateChange.
```

Comment:
I'm not sure I'd want it at StateChange level. Some StateChanges, if not many, do have an absolute default that might not be necessary to remove. I.e. the default BindFBO state change binds to fboId 0, which is the default frame buffer. Would there be a need to dispose it?